pom.xml

```diff
@@ -557,6 +557,27 @@
 						</execution>
 					</executions>
 				</plugin>
+				<plugin>
+					<groupId>org.jacoco</groupId>
+					<artifactId>jacoco-maven-plugin</artifactId>
+					<version>0.8.8</version>
```

Comment:
Why not 0.8.9?

Reply:
Git conflict 😞

Reply:
OK, ik vroeg me af of er misschien een andere reden was, zoals bugs in de nieuwe Jacoco versie!